tests/Functional/Schema/MySQLSchemaManagerTest.php

```diff
@@ -202,18 +202,7 @@ public function testDoesNotPropagateDefaultValuesForUnsupportedColumnTypes(): vo
 
         $diff = $this->schemaManager->createComparator()
             ->diffTable($table, $onlineTable);
-        self::assertNotFalse($diff);
```

Comment:
The expectation that there should be a diff is invalid. It's a bug that is mitigated down the line: https://github.com/doctrine/dbal/blob/a18ffc1a1fde506b17c34e9523fce95f16838359/src/Platforms/AbstractMySQLPlatform.php#L544-L551

We may try removing this condition as well.

Reply:
Funny thing is that this is one of the conditions that cause https://github.com/doctrine/dbal/issues/4747.